internal/pkg/scorecard/test_definitions.go

```diff
@@ -123,29 +123,32 @@ func MergeSuites(suites []TestSuite) ([]TestSuite, error) {
 	mergedSuites := []TestSuite{}
 	for _, suiteSlice := range suiteMap {
 		testMap := make(map[string][]TestResult)
+		logs := ""
 		for _, suite := range suiteSlice {
 			for _, result := range suite.TestResults {
 				testMap[result.Test.GetName()] = append(testMap[result.Test.GetName()], result)
 			}
+			logs = logs + suite.Log
```

Comment:
Would it make sense to add something to separate each suite's logs? 

Maybe something like `logs = logs + fmt.Sprintf("--- %s logs ---\n%s\n\n", suite.Name, suite.Log)`

Reply:
Yeah, that would be clearer

Reply:
Actually, I'm not quite sure. What this function is merging is suites with the same name. The log field is in the SuiteResult field, so we already know which suite the log belongs to. This line is just aggregating the logs of multiple runs of the same suite.  Maybe just separating the logs for each run with a `\n---\n` could help with clarity.